components/contactUs/contactUsMain/ContactUsMain.tsx

```diff
@@ -30,18 +32,23 @@ const submitFormData = async (event: any) => {
   formData.append("your-subject", CONTACT_FORM.DEFAULT_SUBJECT.CONTACT_US);
 
   try {
-    await sendFormRequest(CONTACT_FORM.FORM_IDS.REACH_US, formData, {
+    const result = await sendFormRequest(CONTACT_FORM.FORM_IDS.REACH_US, formData, {
       headers: {
         "Content-Type": "multipart/form-data",
       }
     });
+    alert(result?.data);
   } catch (ex){
+    // eslint-disable-next-line no-console
     console.error(ex);
```

Comment:
```suggestion
    dispatch(errorHandling(true))
```

կամ ոնց էր ճիշտը տենց մի խոսքով